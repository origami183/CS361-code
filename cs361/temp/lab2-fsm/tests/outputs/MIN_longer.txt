NEW.ADMIT -> RDY
RDY.DISPATCH -> RUN
RUN.DISPATCH -> RUN
RUN.SCHEDULE -> RDY
RDY.NIL -> NST
RDY.DISPATCH -> RUN
RUN.NIL -> NST
RUN.EXIT -> TRM
