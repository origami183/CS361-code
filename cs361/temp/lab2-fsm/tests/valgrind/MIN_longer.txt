==1359205== Memcheck, a memory error detector
==1359205== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1359205== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1359205== Command: ../fsm 0 1 1 2 7 1 7 5
==1359205== 
NEW.ADMIT -> RDY
RDY.DISPATCH -> RUN
RUN.DISPATCH -> RUN
RUN.SCHEDULE -> RDY
RDY.NIL -> NST
RDY.DISPATCH -> RUN
RUN.NIL -> NST
RUN.EXIT -> TRM
==1359205== 
==1359205== HEAP SUMMARY:
==1359205==     in use at exit: 0 bytes in 0 blocks
==1359205==   total heap usage: 2 allocs, 2 frees, 8,224 bytes allocated
==1359205== 
==1359205== All heap blocks were freed -- no leaks are possible
==1359205== 
==1359205== For lists of detected and suppressed errors, rerun with: -s
==1359205== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
