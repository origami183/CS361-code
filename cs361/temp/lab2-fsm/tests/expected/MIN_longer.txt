NEW.ADMIT -> RDY
  [Process is live]
RDY.DISPATCH -> RUN
RUN.DISPATCH -> RUN
RUN.SCHEDULE -> RDY
RDY.NIL -> NST
RDY.DISPATCH -> RUN
RUN.NIL -> NST
RUN.EXIT -> TRM

  Process stats:
    runtime: 3
    normal exit: true
